Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -326,9 +330,12 @@ namespace AzToolsFramework::Prefab
 
         // Open all container entities in the new path.
         SetInstanceContainersOpenState(m_rootAliasFocusPath, true);
-        // Set open state on all nested instances in the new focus subtree based on edit scope.
-        SetInstanceContainersOpenStateOfAllDescendantContainers(
-            GetInstanceReference(m_rootAliasFocusPath), m_prefabEditScope == PrefabEditScope::SHOW_NESTED_INSTANCES_CONTENT);
+
+        if (IsPrefabOverridesUxEnabled())
+        {
+            // Set open state on all nested instances in the new focus subtree based on edit scope.
+            SetInstanceContainersOpenStateOfAllDescendantContainers(GetInstanceReference(m_rootAliasFocusPath), true);
```

Comment:
Why was the openState parameter changed from `m_prefabEditScope == PrefabEditScope::SHOW_NESTED_INSTANCES_CONTENT` -> `true`?

Reply:
I notice the condition always returns True when overrides ux flag is enabled, so I explicitly change it to True. And even I change it to True, I am still seeing that the nested container entities are not opened correctly. This could be related to some outliner collapsing issues we discussed recently.

Reply:
```
m_prefabEditScope == PrefabEditScope::SHOW_NESTED_INSTANCES_CONTENT
```

This was added as future-proofing, but it is indeed hardcoded to always be true. If possible, I'd revert it to the check as that is meant to enable switching back and forth between overrides UX and previous UX at runtime.

Reply:
Discussed offline with @AMZN-daimini. This condition can be kept for switching edit scope at runtime, which is useful for future use case on pinning property in DPE. For this into stab, it is okay to have it be deterministic to return True if flag is on.

We agreed that we should bring back this condition into dev in a separate PR so this one can be unblocked.